GetStream/stream-chat-android
# Fix conditions for checking if instance already exists

### 🎯 Goal

Fix conditions for checking if an instance already exists.

### 🛠 Implementation details

Conditions for checking if `StateRegistry.instance` and `LogicRegistry.instance` already exist were incorrect.

### 🧪 Testing

1. Do a fresh app install
2. Sign in as any user
3. Observe that logs are not visible
4. Do the same on `develop`

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
